commands/WirelessCommand/index.js

```diff
@@ -562,32 +570,120 @@ WirelessCommand.prototype.__configure = function __configure(ssid, cb) {
 					return "You must enter a password. Let's try again...";
 				}
 
-			}], manualChoices);
+			}]).then(manualChoices);
 		}
 
 		self.newSpin('Asking the Photon to scan for nearby Wi-Fi networks...').start();
 		retry = setTimeout(start, 1000);
 
 	}
 
+	function enterpriseChoices(ans) {
+		self.prompt([
+			{
+				type: 'list',
+				name: 'eap',
+				message: 'EAP Type',
+				choices: [
+					'PEAP/MSCHAPv2',
+					'EAP-TLS'
+				]
+			},
+			{
+				type: 'input',
+                name: 'username',
+                message: 'Username',
+                when: function(ans) {
+                	return ans.eap == 'PEAP/MSCHAPv2';
```

Comment:
I see these strings repeated - might be a good idea to replace this inline magic values with constants so the linter can check for typos.